plugins/plugin-docker/che-plugin-docker-client/pom.xml

```diff
@@ -52,6 +52,24 @@
             <artifactId>guice</artifactId>
         </dependency>
         <dependency>
+            <groupId>commons-lang</groupId>
+            <artifactId>commons-lang</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.fabric8</groupId>
+            <artifactId>kubernetes-client</artifactId>
+            <version>1.4.26</version>
```

Comment:
move this variable to https://github.com/eclipse/che-dependencies/blob/master/pom.xml